esx_service/utils/vmdk_utils.py

```diff
@@ -109,21 +122,21 @@ def get_datastores():
     'url' is datastore URL (e.g. '/vmfs/volumes/vsan:572904f8c031435f-3513e0db551fcc82')
     'dockvol-path; is a full path to 'dockvols' folder on datastore
     """
-    if datastores == None:
-        init_datastoreCache()
+    init_datastoreCache()
     return datastores
```

Comment:
In this case, even "get_datastores" returns a copy of global variable "datastores" is not right. Since now the copy of "datastores" can be None too.  We should let the init_datastoreCache() to return "datastores". Then funtion "get_datastores" just return "init_datastoreCache()".